test/test_lrf.cpp

```diff
@@ -0,0 +1,33 @@
+#include <gtest/gtest.h>
+
+#include <geolib/sensors/LaserRangeFinder.h>
+
+#include <cmath>
+
+double ANGLE_MIN = -M_PI_2;
+double ANGLE_MAX = M_PI_2;
+uint N_BEAMS = 10;
+
```

Comment:
Perhaps also test some angles within the range of the LRF while we are writing tests.